### ISSUE TITLE
Create a test suite for net module

### ISSUE THREAD

**Issue by mielientiev** [propensive/rapture#204]

**Comment by mielientiev** [comment]
@propensive does `test` module have `before\after` API? I want to use it for running docker container before test cases and kill it after test cases would be finished

**Comment by propensive** [comment]
No, it doesn't. It's quite basic. I think what you've done with ScalaTest is fine for now. There might be an opportunity to improve the API later, but it's not at all important. :)
